e2e/davinci-app/main.ts

```diff
@@ -10,9 +11,9 @@ import protect from './components/protect.js';
 import flowLinkComponent from './components/flow-link.js';
 import socialLoginButtonComponent from './components/social-login-button.js';
 
-const config = {
+const config: DaVinciConfig = {
   clientId: '724ec718-c41c-4d51-98b0-84a583f450f9',
-  redirectUri: window.location.href,
+  redirectUri: window.location.origin + '/',
```

Comment:
because i added query params to the testing, i just appended the '/' since it's stripped when using `origin`.